x-pack/plugin/src/test/resources/rest-api-spec/test/ml/jobs_crud.yml

```diff
@@ -948,21 +830,6 @@
       ml.open_job:
         job_id: jobs-crud-force-close-job
 
-  - do:
```

Comment:
This honestly didn't seem necessary given what the test was testing. Seemed like a bad copy paste in the past.